docs/docsite/rst/playbook_guide/playbooks_filters.rst

```diff
@@ -2201,3 +2201,5 @@ This can then be used to reference hashes in Pod specifications:
        Have a question?  Stop by the google group!
    :ref:`communication_irc`
        How to join Ansible chat channels
+   `Python 3 Regular expression operations <https://docs.python.org/3/library/re.html>`_
+       How to use inline regular expression flags
```

Comment:
It looks like the force push missed removal of one change to the `docs` directory.

Reply:
oh my bad